### PR TITLE
Feature/vocabulary

### PR DESCRIPTION
**Pull request by soderluk**
Update contexts
Add vocabularies
Update core ontology

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -20,4 +20,5 @@ cp -R "${OUT_FOLDER}"/* "${ARTIFACTS}"/
 
 # Copy over the ontologies and contexts to GH pages
 cp -R "${WORKDIR}/ontologies" "${ARTIFACTS}"/
-cp -R "${WORKDIR}/contexts" "${ARTIFACTS}"/
\ No newline at end of file
+cp -R "${WORKDIR}/contexts" "${ARTIFACTS}"/
+cp -R "${WORKDIR}/vocabularies" "${ARTIFACTS}"/
```
Comment (lietu) [inline build.sh]: Please add newline to end of file

**Review by lietu — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -10,4 +10,173 @@ The HTML version of the ontology documentation is hosted at our
...
+    ...
+
+As can be seen from the example, we define the attribute, the title for the
+attribute, description and if the attribute is required or not. We can also
+define the value type of the attribute, e.g. `"dli:valueType": "xsd:object"`.
+If no `valueType` is defined, the default should always be `xsd:string`.
+We can also define default values for the attributes with 
+`"dli:defaultValue": "Placeholder"`,
+as well as different supported values with 
+`"dli:supportedValue": ["Yes", "No", "None"]`.
```
Comment (edanchenkov) [inline README.md:175]: Maybe `values` in plural?

**Review by edanchenkov — APPROVED** [review]
README.md is well done

File: build.sh (hunk 1 above)
Comment (soderluk) [reply build.sh]: Added.

File: README.md (hunk 2 above)
Comment (soderluk) [reply README.md:175]: As with the `supportedClass` and `supportedAttribute`, this is in singular. It's definition in the ontology says what it is, and what it means. I think it's best to stick to singular, as the value can either be a list, or just a single string value.

**Pull request merged**